test/MockSDKInfo.js

```diff
@@ -47,7 +47,7 @@ var MockSDKInfo = (function(window) {
             cordova.interceptExec(SDKINFO_SERVICE, "getInfo", function (successCB, errorCB, args) {
                 successCB(new SDKInfo("4.1.0", 
                                       ["com.salesforce.oauth", "com.salesforce.sdkinfo", "com.salesforce.sfaccountmanager", "com.salesforce.testrunner", "com.salesforce.smartstore", "com.salesforce.smartsync"], 
-                                      "SalesforceHybridTest", "1.0",
```

Comment:
Unrelated to the rest of the PR, just noticed this test is not passing in a browser